waspc/src/Wasp/Analyzer/StdTypeDefinitions.hs

```diff
@@ -41,5 +43,6 @@ stdTypes =
   TD.addDeclType @Route $
   TD.addDeclType @Query $
   TD.addDeclType @Action $
+  TD.addDeclType @Job $
```

Comment:
There is nothing to comment on in this file, I'm disappointed, it's too easy to add a new declaration :D TH is taking our jobs!